packages/core/package.json

```diff
@@ -2,12 +2,17 @@
   "name": "@code-pushup/core",
   "version": "0.49.0",
   "license": "MIT",
+  "description": "Core business logic for the used by the [Code PushUp CLI](https://github.com/code-pushup/cli/tree/main/packages/cli/README.md)",
+  "homepage": "https://github.com/code-pushup/cli/tree/main/packages/core",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/code-pushup/cli.git",
+    "directory": "packages/core"
+  },
   "dependencies": {
     "@code-pushup/models": "0.49.0",
     "@code-pushup/utils": "0.49.0",
     "@code-pushup/portal-client": "^0.9.0",
     "ansis": "^3.3.0"
-  },
```

Comment:
Is this no longer needed?

Reply:
it is overwritten by the build process. in `esbuild.config.js`